src/models/git/FileStatus.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * Represent a git branch.
```

Comment:
not correct

Reply:
done

Reply:
correction not accurate. 
`Represent a file with its git status.` seems more appropriate.

Reply:
done